src/rules/spaceWithinParensRule.ts

```diff
@@ -73,9 +73,13 @@ class SpaceWithinParensWalker extends Lint.AbstractWalker<Options> {
     public walk(sourceFile: ts.SourceFile) {
         forEachToken(sourceFile, (token: ts.Node) => {
             if (token.kind === ts.SyntaxKind.OpenParenToken) {
-                this.checkOpenParenToken(token);
+                if (sourceFile.text.charAt(token.end) !== ts.tokenToString(ts.SyntaxKind.CloseParenToken)) {
+                    this.checkOpenParenToken(token);
+                }
             } else if (token.kind === ts.SyntaxKind.CloseParenToken) {
-                this.checkCloseParenToken(token);
+                if (sourceFile.text.charAt(token.end - 2) !== ts.tokenToString(ts.SyntaxKind.OpenParenToken)) {
```

Comment:
`ts.tokenToString(ts.SyntaxKind.OpenParenToken)` just returns `(`. Consider comparing with the string directly.
Same for CloseParenToken above

Reply:
Thanks for the feedback, I updated to compare directly with the corresponding strings.